.eslintrc.js

```diff
@@ -19,6 +19,8 @@ module.exports = {
     "comma-dangle": "off",
```

Comment:
@trock-broadinst so shouldn't "comma-dangle" be set to "always" here? 

And meta-question: how can we know that our eslint rules are in sync with the prettier formatting?

Reply:
I remember there was a complaint about it before, I'm not sure who from, but you can know the rules are in sync because other than the explicitly rewired ones, all eslint and prettier definitions are declared in `extends: ['airbnb-typescript-prettier'],`

Reply:
But isn't this an example of them _not_ being in sync? We requiring trailing commas via prettier, but here the value is set to "off".

Reply:
that's true, I'll fix it, it has to be enabled/disabled in both, so any rules that will auto-format must be applied to both the checker(eslint) and formatter(prettier)

Reply:
Thank you! And can you make sure that the other eslint settings are in sync with the prettier settings?

Reply:
With the Prettier ESLint plugin and its rule, ESLint requires that code be formatted with Prettier.

https://prettier.io/docs/en/integrating-with-linters.html

Other ESLint formatting rules that overlap with Prettier (such as `max-len`, `comma-dangle`, etc) can either be configured to match Prettier or be disabled, since the Prettier rule enforces formatting.